### DISCUSSION TITLE
HTML code within blockquotes while using the raw HTML editor rendering issues

### DISCUSSION THREAD

**Discussion by Xorbite** [requarks/wiki#6184] [Error / Bug Report]
As of lately, I have been trying to add some HTML code within a HTML coded page using the raw HTML editor of course. The issue is that when I try adding that HTML code within the blockquotes element to explain its functions in the wiki page, wikijs will think that the included HTML code within the blockquotes element is also code which should be used in the rendering process and therfore it will not know what to do afterwards and instead show a blank page. 

The html code within the blockquotes element should not act as a part of the main page HTML code but instead as just an example that has nothing to to do with the main layout, yet based on its behavior it doesn't seem to understand that it should ignore the code within the blockquotes element. 

How do I fix that behavior? I really need to the use the raw HTML editor for what I want to make and so I can't use any other editor modes at the moment as they don't support implementing iFrames to show how a specific component works. Thanks in advance!

**Comment by NGPixel** [comment]
Use the markdown editor instead? You can insert any HTML in this editor, alongside Markdown for code blocks.

**Reply by Xorbite** [reply]
Well, the thing is, that I really need to use the HTML editor to achieve the behaviour I'm looking for (I have some cutom animations and effects implemented, like the "sticky" property applied to a couple of iframe elements)

**Reply by Xorbite** [reply]
I'm making an internal wiki system for a company, the wiki system is intended for devs to be able to work around with the different AngularJs components. Each component is implemented using iFrames (unless if you have a better way to do so) and those iFrames of each component has its own behaviour and its own animations of course. 

The IFrame elements will stay sticky when scrolled into and will stay so until you reach another IFrame element. That's basically a brief explanation of the system should do. Using the MD editor as you've mentioned will not give me the freedom to do such animations, or even style everything as I would like. 

I recently stumbled upon WikiJs and saw its potential and honestly, you have done a great job making this. It does have its limitations, but for the most part it does the job, and it does well enough. Just a couple bugs here and there but otherwise, a really great system overall, I must admit

**Reply by Xorbite** [reply]
Oh I see what you mean actually, the RM editor has support for HTML, CSS and JS as well. That's actually what I was looking for, thanks a lot man.

**Reply by Xorbite** [reply]
Although, I just don't think that the HTML editor should behave like that
